models/dpr_test/modelparameters.m

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
I'd like to see the soc_system.dts file added, especially with this being a reference design so that people can see what that looks like.

Reply:
Yeah, good point.